src/shapes/optix/mesh.cuh

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#include <mitsuba/render/optix/common.h>
+
+struct OptixMeshData {
+    const optix::Vector3u *faces;
+    const optix::Vector3f *vertex_positions;
+    const optix::Vector3f *vertex_normals;
+    const optix::Vector2f *vertex_texcoords;
+};
+
+#ifdef __CUDACC__
+
+/// Read in an array of Array<T,3> with a true stride of 3 (and not 4, see
```

Comment:
Is this still relevant?

Reply:
Yes as the mesh data buffers (e.g. `vertex_positions`) are stored without alignment (3 floats per point and not 4 floats).